examples/spawn-and-move/abis/base/base.json

```diff
@@ -0,0 +1,98 @@
+[
```

Comment:
Hm I think we can ignore the base abi. I don't think it is useful for the user

Reply:
agree with removing "base contract"'s abi.

> I also think we can avoid outputting abis for base generally and only output them for deployed contracts. What do you think?

yeah if we are not going to need them we should remove it, i was not sure which are required and which are not.

so we mainly need abi for `world` and user defined `DojoContract`s only correct?